### ISSUE TITLE
Alexa Integration with Kodi (SPMC)?

### ISSUE THREAD

**Issue by fonzie24** [koying/SPMC#494]
I followed this project's instructions: https://github.com/m0ngr31/kodi-alexa

but I have not been able to get my Amazon Echo Dot's Alexa to communicate with my SPMC machine.  If I send navigation commands, I get no response from SPMC and if I give a command to open a specific movie or browse my database I get a "error parsing" message.

Then it occured to me that it may be due to the fact that I'm using SPMC and not the official Kodi.  Looking at the steps inolved to get this up and running do you think that is a possibility?  Or are SPMC and Kodi so similar that the same setup should work across both systems?

I really hope I can get it running on SPMC because I definitely prefer all the tweaks that come along with it compared to vanilla Kodi.

Sorry if this is the wrong place to ask, but I couldn't find anywhere else that was appropriate for this to be posted since the thread was locked.

Thanks in advance.

**Comment by nikotime** [comment]
Hey Fonzie24

It will be my weekend task to try and get Alexa and SPMC talking together - did you have any more luck in getting SPMC talking to Alexa?

**Comment by fonzie24** [comment]
Yes I got it working by scraping my failed attempt and starting this guide from the beginning.  Not sure where I messed up, but it does fully work with SPMC just as it should with Kodi.

**Comment by nikotime** [comment]
That's excellent - thank you very much for letting me know.

**Comment by Njur** [comment]
Hi guys. Did you get this working on the Shield?

**Comment by fonzie24** [comment]
I have my Amazon Echo dot controlling my Kodi installation on my first generation Nvidia Shield TV.

**Comment by Njur** [comment]
Okay thanks for the feedback. I can order it then. But doesn't it only work with Kodi in new version, not spmc? 
I have 1 gen shield also

**Comment by fonzie24** [comment]
I had originally set it up with SPMC before the new version of Kodi 17.1 had been released.  So yes, it works with SPMC as far as I know.
